packages/vital-devices-react-native/src/model/scanned_device.ts

```diff
@@ -1,7 +1,30 @@
 import type { DeviceModel } from './device_model';
+import type { QuantitySample } from '@tryvital/vital-core-react-native';
 
 export interface ScannedDevice {
   id: string;
   name: string;
   deviceModel: DeviceModel;
 }
+
+export enum Libre1SensorState {
+  Unknown = "unknown",
+  NotActivated = "not_activated",
+  WarmingUp = "warming_up",
+  Active = "active",
+  Expired = "expired",
+  Shutdown = "shutdown",
+  Failure = "failure",
```

Comment:
use string literal union please
`type Libre1SensorState = "unknown" | "not_activated" | ...;`

Reply:
we are passing strings over interop, and enum requires fixing it up to the enum type.

Reply:
Oh I see!